lib/gui/app/components/progress-button/progress-button.tsx

```diff
@@ -44,7 +44,7 @@ const FlashProgressBar = styled(ProgressBar)`
 `;
 
 interface ProgressButtonProps {
-	type: 'decompressing' | 'flashing' | 'verifying';
+	type: FlashState['type'];
```

Comment:
This is not equivalent as type is nullable in FlashState

Reply:
@zvin what's the issue with it? We should avoid re-defining types we already have - FlashState's `type` is the one we should use for the UI and it extends the one from etcher-sdk's flash progress state since it can also be `downloading`, which isn't a concept of the sdk

Reply:
@zvin what's the issue with it though? This just streamlines the type so it's not defined in multiple places